File: 백준/Bronze/11720. 숫자의 합/숫자의 합.cc
# include <iostream>
# include <string>
using namespace std;
 
int main(){
    
   int N;
   cin>>N;//숫자 개수 
   
   string s;
   
   int sum=0;
   cin>>s;
   
   for(int i=0;i<N;i++){
   sum+=s[i]-'0';//char형을 int 형으로 변환 
   }
  
  cout<<sum;
   return 0; 
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());// 문자열 입력 받기 
        String sNum = br.readLine();// 문자열 입력 받기 
        
        char[] cNum=sNum.toCharArray();
        int sum=0;
        
        for(int i=0;i<cNum.length;i++){
            sum+=cNum[i]-'0'; //아스키코드값 48를 빼줘야 char를 int로 변환가능
        }
        
        System.out.println(sum);
    }
}